### ISSUE TITLE
Fold headers don't render ligatures at end of line

### ISSUE THREAD

**Issue by alexozer** [Microsoft/vscode#38451]
- VSCode Version: 1.18.0
- OS Version: Arch Linux (latest)

Steps to Reproduce:

1. Use a font with ligatures (like Fira Code) and enable ligatures in settings
2. Enter this into a buffer:

```python
def func(): # blah ->
    stuff()
    things()
    something()
    anything()
```

3. Click the minus button to the left of the first line to fold the line
4. Observe the "->" ligature at the end of the line unrender

Reproduces without extensions: Yes

Sidenote: I acknowledge it's probably a pretty low-priority bug.

**Comment by roblourens** [comment]
I can't repro, can you include a text snippet?

**Comment by alexozer** [comment]
@roblourens sure:

```python
def func(): // blah ->
    stuff()
    things()
    something()
    anything()
```

Clicking the minus to the left of the first line causes the "->" ligature to unrender for me.

**Comment by roblourens** [comment]
I can repro with that one, thanks.

**Comment by alexandrudima** [comment]
Already fixed via 2217a39b3ee4c1ebf0516a5f52d5561a6825778c
